src/NServiceBus.Core/Sagas/IHandleSagaNotFound.cs

```diff
@@ -1,20 +1,18 @@
 #nullable enable
 
-namespace NServiceBus.Sagas;
+namespace NServiceBus;
```

Comment:
I kept the same name but changes namespaces since I think NServiceBus is ok since end user code needs to implement it.

That said I think this might be confusing and renaming to `ISagaNotFoundHandler` might be better. Thoughts?